src/traccuracy/metrics/_track_overlap.py

```diff
@@ -57,11 +59,18 @@ class TrackOverlapMetrics(Metric):
 
     """
 
-    supports_many_to_one = True
-
     def __init__(self, include_division_edges: bool = True):
         self.include_division_edges = include_division_edges
 
+    def _validate_matcher(self, matched: Matched) -> bool:
```

Comment:
I went down an issue rabbit hole and I believe that it is correct that track overlap works with all matchers. But, perhaps we should double/triple check? Or, just reassure me that you already did so 😄

Reply:
My due diligence was limited, but I think based on what I've ready that all of our current matchers should be ok. I'm going to add to the docstring so that we have a note that the intention si to be many to one.